Sample_v3/Samples/SimpleChat/Views/ComposerView.swift

```diff
@@ -31,38 +27,36 @@ class ComposerView: UIView {
             }
         }
     }
+
     var calculatedHeight: CGFloat {
         if isKeyboardShown {
-             return textView.contentSize.height + 20
+            return textView.contentSize.height + 20
         } else {
             return textView.contentSize.height + 20 + (window?.safeAreaInsets.bottom ?? 0)
         }
     }
     
     override var intrinsicContentSize: CGSize {
-        return CGSize(width: super.intrinsicContentSize.width, height: calculatedHeight)
+        CGSize(width: super.intrinsicContentSize.width, height: calculatedHeight)
     }
     
     override func awakeFromNib() {
         super.awakeFromNib()
         translatesAutoresizingMaskIntoConstraints = false
         
-        NotificationCenter.default.addObserver(
-            self,
-            selector: #selector(handle(keyboardShowNotification:)),
-            name: UIResponder.keyboardWillShowNotification,
-            object: nil
-        )
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(handle(keyboardShowNotification:)),
+                                               name: UIResponder.keyboardWillShowNotification,
+                                               object: nil)
```

Comment:
@buh , why is this happening? I thought you fixed it in #410 ?

Reply:
This is a different formatter parameter.

Reply:
We can fix it here if you want.

Reply:
I left it like with for arguments, because we use it mostly in this way, but sometimes of course it's not so good. But I think I had to make it optional to use both styles.

Reply:
I see, so we allow this format for function declarations but not for the situation where we call it? I'd suggest using the style as it was originally here as the preferred one.